lib/ruby_lsp/listeners/completion.rb

```diff
@@ -169,6 +171,11 @@ def on_call_node_enter(node)
           end
         end
 
+        if ["(", ","].include?(@trigger_character)
+          complete_keyword_arguments(node)
+          return
+        end
```

Comment:
Instead of checking on call nodes if the trigger character matches, we should start handling the event for `Prism::ParametersNode`, so that completion is triggered for parameters.

This should be a matter of 

1. [Registering the event](https://github.com/Shopify/ruby-lsp/blob/23a7084fe44636c140dee3a9d54252f920fcabcb/lib/ruby_lsp/listeners/completion.rb#L94)
2. [Adding the node as a possible target](https://github.com/Shopify/ruby-lsp/blob/23a7084fe44636c140dee3a9d54252f920fcabcb/lib/ruby_lsp/requests/completion.rb#L59)
3. Creating an `on_parameters_node_enter` handler
4. Doing the work to provide keyword completion inside that method